src/pages/StudioPage.js

```
@@ -0,0 +1,339 @@
+            </span>
+          </>
+        ) : (
+          "Link Canva"
+        )}
+      </button>
+    </div>
+  );
+};
+
+// Mint NFT button and status display
+import React, { useState } from "react";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)